android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1569,6 +1569,7 @@ public void onPlayerError(PlaybackException e) {
                 break;
         }
         eventEmitter.error(errorString, e, errorCode);
+        youboraPlugin.getAdapter().fireFatalError("ExoPlayer", errorCode, errorString);
```

Comment:
A couple of points here:
- youbora already has an integration with the player error events, why are we doing a direct integration here?
- Not all errors are fatal.

Reply:
See https://bitbucket.org/npaw/exoplayer2-adapter-android/src/ae7cc4fa36ab711d0e09f7bf1e6d69a25d7cbd11/exoplayer2-adapter/src/main/java/com/npaw/youbora/lib6/exoplayer2/Exoplayer2Adapter.kt#lines-212 .

Reply:
I pointed out in the stand up this current override is not working, because last version update the exception type is changed, we can simply send the youbora by our self and add the updated error code inside

Reply:
This reports the wrong events though. Not everything is a fatal error.

Reply:
plus, we should at least add a TODO in the code explaining WHY we having to do this, the first thing someone reading this code would ask is why youbora's own plugin isn't working.

Like most of our player changes, this feels like a quick hack rather than a proper solution.

Reply:
Only behind window error it's not fatal and in exo player it will also deal in separate function, we should update youbora plugin version to pair with exoplayer, but they are going to update/switch the player so we just fix it for now